beamer/l1_resolution.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
+
+from hexbytes import HexBytes
+
+from beamer.typing import URL
+
+RELAYER_EXECUTABLE = "beamer-relayer"
+
+
+def run_relayer(l1_rpc: URL, l2_rpc: URL, privkey: str, tx_hash: HexBytes) -> None:
+    subprocess.run(
+        [
+            RELAYER_EXECUTABLE,
+            "--l1rpcprovider",
+            l1_rpc,
+            "--l2rpcprovider",
+            l2_rpc,
+            "--l1wallet",
+            privkey,
```

Comment:
Is `privkey` really the private key itself? If so, we should reconsider other ways of providing that data to the relayer, for example via environment variables. Command line arguments are trivially visible.

Reply:
Yes, we need to pass this information to the relayer. What's the recommended approach for this?

Reply:
An environment variable, for example, would be an easy way.

Reply:
I created an issue to discuss and decide this. https://github.com/beamer-bridge/beamer/issues/650